BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditItemProcessor.swift

```diff
@@ -311,8 +334,28 @@ final class AddEditItemProcessor: StateProcessor<// swiftlint:disable:this type_
             state.cardItemState.expirationYear = year
         case let .toggleCodeVisibilityChanged(isVisible):
             state.cardItemState.isCodeVisible = isVisible
+            if isVisible {
+                let cipherId = state.cipher.id
+                Task {
+                    await services.eventService.collect(
+                        eventType: .cipherClientToggledCardCodeVisible,
+                        cipherId: cipherId,
+                        errorReporter: services.errorReporter
+                    )
+                }
```

Comment:
🎨 Similar to my other comment, I wonder if it makes sense to add a helper method in the processor that logs an event.

Reply:
With no longer having `errorReporter` at the call site, a helper method would basically just abstract out the `Task` aspect of it, and I'm not sure how much I like that 🤔